scripts/lib/mkcloud-libvirt.sh

```diff
@@ -57,36 +57,85 @@ function libvirt_net_start()
         ip link set mtu 9000 dev $dev
     done
 
+    boot_mkcloud=/etc/init.d/boot.mkcloud
+    boot_mkcloud_d="$boot_mkcloud.d"
+    boot_mkcloud_d_cloud="$boot_mkcloud_d/$cloud"
+
     if [ -z "$NOSETUPPORTFORWARDING" ] ; then
+        # FIXME: hardcoded assumptions about admin net host range
         nodehostips=$(seq -s ' ' 81 $((80 + $nodenumber)))
```

Comment:
No that's wrong - for a start, this range excludes the admin server which typically (but not always) has IP ending `.10`. It's *still* not 100% clear exactly what you mean, but I think what you are maybe trying to say is that we should try to avoid forwarding ports to nodes which don't use them? But this PR already makes changes in that direction: see 7335aacc07f6aa6fcbd0dcdc7c52d3c741f1704a.  Yes we could trim the forwarding even further, but not without making the code a lot more complex.